includes/class-logger.php

```diff
@@ -45,21 +45,7 @@ class Logger {
 	 *     'debug': Debug-level messages.
 	 */
 	public static function log( $message, $level = 'info' ) {
-		if ( ! self::can_log() ) {
-			return;
-		}
-
-		self::init_logger();
-		self::$logger->log( $level, $message, [ 'source' => self::LOG_FILENAME ] );
```

Comment:
Now that this line is gone, `LOG_FILENAME` is not needed in this class.

Reply:
Removed